### ISSUE TITLE
Tests in skimage/morphology/tests/test_grey.py use lena

### ISSUE THREAD

**Issue by sergiopasra** [scikit-image/scikit-image#1865]
To remove the lena image (#1855) some tests have to be rewritten to use a different image.
The following tests in ``skimage/morphology/tests/test_grey.py`` use lena

* All tests in class TestMorphology

**Comment by OrkoHunter** [comment]
In the [`TestMorphology`](https://github.com/scikit-image/scikit-image/blob/master/skimage/morphology/tests/test_grey.py#L18-L66) class of the test, lena is being passed from all the individual tests to the function `morph_worker` as the argument `img` and is *never* being used. Doen't the argument `img` seem useless?

```py
def morph_worker(self, img, fn, morph_func, strel_func):                      
    matlab_results = np.load(os.path.join(data_dir, fn))
    k = 0
    for arrname in sorted(matlab_results):
        expected_result = matlab_results[arrname]
        mask = strel_func(k)
        actual_result = morph_func(lena, mask)
        testing.assert_equal(expected_result, actual_result)
        k = k + 1 
     
```

**Comment by soupault** [comment]
Closed via #1985
